CODEOWNERS

```diff
@@ -5,6 +5,9 @@ cardano-cli/README.md                                         @Jimbo4350 @newhog
 # Release manager added for notification on dependency updates
 cabal.project                                                 @Jimbo4350 @newhoggy @carbolymer
 
+# General reviewers per PR
+#                        Duncan   Jordan     Erik   John      Mateusz     Kevin          Carlos             Sam
+*                        @dcoutts @Jimbo4350 @erikd @newhoggy @carbolymer @kevinhammond @CarlosLopezDeLara @disassembler
```

Comment:
I've copied this part from `cardano-api`

Reply:
You can remove dcoutts, erik and carlos.